erfid.rb

```diff
@@ -48,17 +61,23 @@ def error(message)
   exit(-1)
 end
 
-def display_success
-  @green_led ||= PiPiper::Pin.new(:pin => 23, :direction => :out)
-  3.times do
-    @green_led.on
-    sleep(0.5)
-    @green_led.off
+def display_success(status)
+  if status == "RFID sign in" || status == "RFID exist"
```

Comment:
`|| status == "RFID exist"` should go away when the staff_dashboard PR gets merged